scst-scan/verify-app-scanning.hbs.md

```diff
@@ -51,3 +51,31 @@ To retrieve a vulnerability report:
    imgpkg pull -b $SCAN_RESULT_URL -o scan-results/
    ls scan-results/
    ```
+
+## <a id="validating-scan-format"></a> Validating Scan Format
+
+After retrieving the scan results, the scan results can be validated via this CycloneDX tool, [sbom-utility](https://github.com/CycloneDX/sbom-utility). This tool is designed to validate CycloneDX and SPDX BOMs against versioned schemas.
+1. Setup and install using the instructions [here](https://github.com/CycloneDX/sbom-utility#installation).
+2. Run the `sbom-utility` CLI with the subcommand [validate](https://github.com/CycloneDX/sbom-utility#validate) to validate the scan report against its declared format (e.g., SPDX, CycloneDX) and version (e.g., "2.2", "1.4", etc.).
```

Comment:
Can we put a table here that specifies supported version?  SPDX 2.2 and Cyclone DX 1.2/1.3./1.4?  We should tell the user here what formats to validate against that AMR Observer supports and not leave them guessing.

Reply:
Probably makes sense for the table to be here:

https://github.com/pivotal/docs-tap/pull/2647/files#r1242626833

Reply:
I could put a table however the `sbom-utility` tool itself does provide a CLI command to show what schemas/version/format are available (https://github.com/CycloneDX/sbom-utility#schema). 

```
lrobin@lrobin6MD6R sbom-utility-v0.11.0-darwin-amd64 % ./sbom-utility schema list
Welcome to the sbom-utility! Version `v0.11.0` (sbom-utility) (darwin/amd64)
============================================================================
[INFO] Loading license policy config file: `license.json`...
[INFO] Scanning document for vulnerabilities...
name                          format     version   variant      file                                             url
----                          ------     -------   -------      ----                                             ---
CycloneDX v1.5 (development)  CycloneDX  1.5       development  schema/cyclonedx/1.5/bom-1.5-dev.schema.json     https://raw.githubusercontent.com/CycloneDX/specification/v1.5-dev/schema/bom-1.5.schema.json
CycloneDX v1.4                CycloneDX  1.4       (latest)     schema/cyclonedx/1.4/bom-1.4.schema.json         https://raw.githubusercontent.com/CycloneDX/specification/master/schema/bom-1.4.schema.json
CycloneDX v1.4 (custom)       CycloneDX  1.4       custom       schema/test/bom-1.4-custom.schema.json
CycloneDX v1.3                CycloneDX  1.3       (latest)     schema/cyclonedx/1.3/bom-1.3.schema.json         https://raw.githubusercontent.com/CycloneDX/specification/master/schema/bom-1.3.schema.json
CycloneDX v1.3 (custom)       CycloneDX  1.3       custom       schema/test/bom-1.3-custom.schema.json
CycloneDX v1.3 (strict)       CycloneDX  1.3       strict       schema/cyclonedx/1.3/bom-1.3-strict.schema.json  https://raw.githubusercontent.com/CycloneDX/specification/master/schema/bom-1.3-strict.schema.json
CycloneDX v1.2                CycloneDX  1.2       (latest)     schema/cyclonedx/1.2/bom-1.2.schema.json         https://raw.githubusercontent.com/CycloneDX/specification/master/schema/bom-1.2.schema.json
CycloneDX v1.2 (strict)       CycloneDX  1.2       strict       schema/cyclonedx/1.2/bom-1.2-strict.schema.json  https://raw.githubusercontent.com/CycloneDX/specification/master/schema/bom-1.2-strict.schema.json
SPDX v2.3                     SPDX       SPDX-2.3  (latest)     schema/spdx/2.3/spdx-schema.json                 https://raw.githubusercontent.com/spdx/spdx-spec/development/v2.3/schemas/spdx-schema.json
SPDX v2.3.1 (development)     SPDX       SPDX-2.3  development  schema/spdx/2.3.1/spdx-schema.json               https://raw.githubusercontent.com/spdx/spdx-spec/development/v2.3.1/schemas/spdx-schema.json
SPDX v2.2.2                   SPDX       SPDX-2.2  (latest)     schema/spdx/2.2.2/spdx-schema.json               https://raw.githubusercontent.com/spdx/spdx-spec/v2.2.2/schemas/spdx-schema.json
SPDX v2.2.1                   SPDX       SPDX-2.2  2.2.1        schema/spdx/2.2.1/spdx-schema.json               https://raw.githubusercontent.com/spdx/spdx-spec/v2.2.1/schemas/spdx-schema.json
```

There's also a way to add more schemas manually with a config file https://github.com/CycloneDX/sbom-utility#adding-sbom-formats-schema-versions-and-variants. 

I think we can tell the user they can run the `./sbom-utility schema list` command to show the supported versions/format/schemas. What do you think?